diagram-viewer/src/main/java/com/powsybl/diagram/viewer/MainViewController.java

```diff
@@ -302,6 +309,20 @@ private void initSubstationsTree(Network network) {
         vlTree.setShowRoot(true);
     }
 
+    private boolean testComponent(IdentifiableType type, Identifiable<?> identifiable) {
+        boolean result = false;
+        if (type == IdentifiableType.NETWORK || type == IdentifiableType.SUBSTATION || type == IdentifiableType.VOLTAGE_LEVEL) {
```

Comment:
ok for network and voltage level, but I would expect that choosing substation removes the voltage levels which have no substation (directly under network)